### DISCUSSION TITLE
What is the app missing?

### DISCUSSION THREAD

**Discussion by Jeffser** [Jeffser/Alpaca#96] [Q&A]
Hi guys, I wanted to ask you all for what you want to see added to the app the most, [these](https://github.com/Jeffser/Alpaca/milestone/1) are the features I will (probably) add for 1.0.0

Is there something else you guys want to see added to the app? it doesn't matter how wild the ideas are I want to keep developing Alpaca.

Also, if you guys want to see a change on something that already exists, please tell me, thanks

**Comment by loulou64490** [comment]
I proposed this https://github.com/Jeffser/Alpaca/issues/70

**Reply by Jeffser** [reply]
Oh yeah I forgot 💀

**Comment by Quantumdit** [comment]
Thanks for making this great app!

In the "Manage Models" menu, I'd appreciate the following features:
1. A search bar to quickly narrow down to the models I'm interested in (especially since the models aren't in alphabetical order).
2. Some way of seeing more information about the model. For example, when I was looking at the llama3 model, it didn't seem to show whether it's the 8B parameter or 70B parameter version.

Thanks! 😊

**Reply by Jeffser** [reply]
Hi, yes I can add the search bar. About the other thing, once you press the download button it gives you the option to select a tag, so that's where it says 8B or 70B

**Reply by Quantumdit** [reply]
Doh, I'm a dummy. 🤦‍♀️

Thanks for the response.

**Reply by Jeffser** [reply]
the searchbar is ready for the next release

**Reply by Quantumdit** [reply]
Awesome, thanks!

**Comment by TylerLaBree** [comment]
I'm always too lazy to edit chat titles, so auto-generating them could be really nice: https://github.com/Jeffser/Alpaca/issues/45

**Reply by Jeffser** [reply]
This is ready for the next big release !

**Comment by loulou64490** [comment]
Hey @Jeffser, idk if you're still on the project, but it would be nice to add the [Crowdin](https://crowdin.com/) integration for translation, it would more efficient and easier for all people to translate the app ! Thanks

**Reply by Jeffser** [reply]
This seems like too much work honestly hahaha

**Comment by nan-value** [comment]
The ability to change how much CPU power you are willing to use at a given time. Personally the app only reaches about 55% utilization on my machine and I think that extra 30% that arent getting used would make a massive difference for me. Its not hard to imagine a situation where doing the inverse and running with small overhead while you are working primarily on something more intensive could easily benefit from the ability to choose how much resources you are willing to throw at it

**Reply by Jeffser** [reply]
Hi, there's nothing in the app itself that limits the CPU power, there might be something going on with your setup that makes Alpaca / Ollama only use 55% or maybe you have a really good CPU and Ollama can't use it all.

**Comment by angel-tech1** [comment]
Change chat font size.

**Comment by niekdomi** [comment]
A search function to search for chats and search for chat content

**Comment by Melechtna** [comment]
An optional method to allow the models to perform web searches to find information. If opt in, I don't see why this would be an issue.
